CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] (beta)
 
+### Added
+- Groups and Teams service support!  Channel messages and Files are now available for backup and restore in the CLI: `corso backup create groups --group '*'`
+  * The cli commands for "groups" and "teams" can be used interchangably, and will operate on the same backup data.
+  * Restore and Export is available for channel files.  Channel messages are available for Export, but not Restore.
```

Comment:
2 caveats:
- Export for files is still pending
- As of now only files from public channels are backed up (ie root site)

Reply:
These caveats are ok but we should do the following:
1) `Groups and Teams service support!` -> Call it Preview. CLI commands should provide the preview message also.
2) Remove Export.
3) Add the caveats to "Known issues" and limitations. The limitations being: Groups support is in preview and not complete. Files are only backed up from public channels. Export is not yet availbable.

Reply:
> Remove Export
Sharepoint export, you mean?  Or both sharepoint and messages?